### DISCUSSION TITLE
Run `pnpm install` in a nested package inside a workspace that doesn't belong to the workspace

### DISCUSSION THREAD

**Discussion by rdsedmundo** [pnpm/pnpm#3427] [Q&A] [answered]
I have this structure:

```
repo/
    package.json
    .pnpm-workspace.yaml (packages: packages/**)
    packages/
        a/
            dist/
                package.json
                .npmrc (recursive-install=false)
        b/
        c/
        d/
        e/
```

I have a scenario where A imports B and C and bundles them, cherry-picking only the dependencies those three packages require. This creates a  `dist/` folder with a new `package.json`. I want to make an artifact of only those dependencies, thus I run `pnpm install` inside the `bundled` folder, but I get: `No projects matched the filters in "repo"` (referring to the fact that `dist/` is outside the workspace), but `npm` and `yarn` understands this pretty much fine, they just see that as an standalone package, whereas `pnpm` seems to always try to go through the ancestors and find a workspace definition and match it, which I don't think makes much sense.

Am I missing something here, or is this intended behavior? If yes, is there a better workaround than temporarily renaming the workspace file or moving the `bundled` folder outside the workspace scope?

I even tried to add that `.npmrc` file there with a `recursive-install=false` to see if it would help, but it didn't have effect, it always tries to refer to the workspace unless the file yaml is not there.

Thanks.

**Comment by zkochan** [comment, marked as answer]
The only workaround currently is to create an empty `pnpm-workspace.yaml` file inside the `budled` folder.
